src/test/java/catalinaTema5/Heroku/Table.java

```diff
@@ -0,0 +1,24 @@
+package catalinaTema5.Heroku;
+
+import catalinaTema5.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Table {
+
+     public static void Table() {
```

Comment:
You're missing main method here in order to be able to run the methods from this class.
I was referring in the camelCase comment to change the name of your method from Table() to table()